README.md

```diff
@@ -42,7 +42,7 @@ Please see the separate [sdk-packages](https://github.com/git-for-windows/sdk-pa
 
 ## Getting in contact
 
-We now have a [mailing list](https://groups.google.com/group/git-win-sdk) for developers.
+We now have a [mailing list](https://groups.google.com/group/git-win-sdk) for developers.  Issues should be reported on the [git-for-windows/git tracker](https://github.com/git-for-windows/git/issues); mentioning "SDK" in the subject line is advisable.
```

Comment:
Isn't setting labels a privileged operation?  Anyway, my goal was just that this README provide some guidance on where to report issues; maybe it'd be simpler if you would write your own text, after investigating whether outsiders can actually be permitted to set the "sdk" label?